packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -628,25 +645,23 @@ describe('plugin-memberships', function () {
 
         return webex.memberships.listen()
           .then(() => {
-            debug(`Setting room: ${room.title} to hidden..`);
-            room.isRoomHidden = true;
+            debug(`Hiding my membership in 1-1 space with ID: ${roomId}...`);
+            membership.isRoomHidden = true;
 
-            return webex.rooms.update(room)
-              .catch((reason) => {
-                debug('Failed to hide room: ', reason);
-                assert.fail('Updating room to hidden failed');
-              })
-              .then(async (r) => {
-                debug('room hidden');
-                debug(r);
+            return webex.memberships.update(membership)
+              .then(async (m) => {
+                debug('memberships.update() request returned OK.  Waiting for memberships:updated event');
                 const event = await updatedEventPromise;
 
-                validateMembershipEvent(event, membership, actor);
-              });
+                assert((m.isRoomHidden === event.data.isRoomHidden === true),
```

Comment:
You seem to be combining two assertions into one here. Could you split these into two lines?
```js
assert.isTrue(m.isRoomHidden)
assert.isTrue(event.data.isRoomHidden)
```
That way, if an assertion fails we know which one was the culprit.

Reply:
OK